pkg/binary/envoy/debug/lsof.go

```diff
@@ -55,75 +56,59 @@ func EnableOpenFilesDataCollection(r *envoy.Runtime) {
 // retrieveOpenFilesData writes statistics of open files associated with envoy instance(s) to a json file
 // if succeeded, return nil, else return an error instance
 func retrieveOpenFilesData(r binary.Runner) error { //nolint:gocyclo
```

Comment:
Please split the function into smaller ones rather than turn off linting.

Reply:
Alright, the lint now passes I remove the for loops in previous commits